src/intercept_desc.c

```diff
@@ -545,6 +545,89 @@ crawl_text(struct intercept_desc *desc)
 	intercept_disasm_destroy(context);
 }
 
+/*
+ * get_selinux_enforced
+ * Identifies if SELinux is active on the system.
+ */
+static int
+get_selinux_enforced(void)
+{
+	static int selinux_enforced = -1;
```

Comment:
Given that this function will only be called once I don't see a reason to cache the result like this.